DragaliaAPI.Database/Repositories/FortRepository.cs

```diff
@@ -38,6 +35,11 @@ ILogger<FortRepository> logger
             x => x.DeviceAccountId == this.playerDetailsService.AccountId
         );
 
+    public IQueryable<DbFortDetail> Details =>
+        this.apiContext.PlayerFortDetails.Where(
+            x => x.DeviceAccountId == this.playerDetailsService.AccountId
```

Comment:
When searching for a single entity based on its primary key, it is usually better to use FindAsync, as that will avoid a trip to the DB if it's already being tracked